src/qml/FavoriteChannels.qml

```diff
@@ -86,7 +86,7 @@ Item {
     Component {
         id: favoriteByAccountComponent
         Settings {
-            objectName: "settings_" + category
+            //objectName: "settings_" + category
```

Comment:
In general I advise not to commit commented out code into the final production branch. If you are not sure this is needed or not, you are not ready for review :) Commented code will bitrot over time, in 6 months nobody knows whats with that, and so its ok to just remove it. We can always revert if needed.